modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml

```
@@ -1,741 +1,748 @@
+                  maxmindEditionIDs:
+                    description: |
+                      Список ревизий баз данных, которые будут скачаны при старте.
+
+                      [Подробнее...](https://dev.maxmind.com/geoip/geolite2-free-geolocation-data)
+              legacySSL:
+                description: |
+                  Определяет, разрешены ли устаревшие версии протокола TLS и алгоритмов шифрования (cipher suites).
+
+                  Работа разрешена для следующих версий протокола TLS: `TLSv1`, `TLSv1.1`, `TLSv1.2`, `TLSv1.3`.
+
+                  Допустимые комбинации алгоритмов шифрования, расположенные в порядке от наиболее надёжной к наименее надёжной: `ECDHE-ECDSA-AES256-GCM-SHA384`, `ECDHE-RSA-AES256-GCM-SHA384`, `ECDHE-ECDSA-CHACHA20-POLY1305`, `ECDHE-RSA-CHACHA20-POLY1305`, `ECDHE-ECDSA-AES128-GCM-SHA256`, `ECDHE-RSA-AES128-GCM-SHA256`, `ECDHE-ECDSA-AES256-SHA384`, `ECDHE-RSA-AES256-SHA384`, `ECDHE-ECDSA-AES128-SHA256`, `ECDHE-RSA-AES128-SHA256`, `ECDHE-ECDSA-AES256-SHA`, `ECDHE-RSA-AES256-SHA`, `ECDHE-ECDSA-AES128-SHA`, `ECDHE-RSA-AES128-SHA`, `AES256-GCM-SHA384`, `AES128-GCM-SHA256`, `AES256-SHA256`, `AES128-SHA256`, `AES256-SHA`, `AES128-SHA`.
+
+                  **По умолчанию** включены только `TLSv1.2`, `TLSv1.3` и самые новые алгоритмы шифрования.
+              disableHTTP2:
+                description: |
+                  Выключить ли HTTP/2.
+              config:
+                description: |
+                  Секция настроек Ingress-контроллера, в которую в формате `ключ: значение(строка)` можно записать [любые возможные опции](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/).
+
+                  > **Внимание.** Ошибка в указании опций может привести к отказу в работе Ingress-контроллера.
+
+                  > **Внимание.** Не рекомендуется использовать данную опцию, так как не гарантируется обратная совместимость или работоспособность Ingress-контроллера.
+              additionalHeaders:
+                description: |
+                  Дополнительные header'ы, которые будут добавлены к каждому запросу. Указываются в формате `ключ: значение(строка)`.
+              additionalLogFields:
+                description: |
+                  Дополнительные поля, которые будут добавлены в логи nginx. Указываются в формате `ключ: значение(строка)`.
+              resourcesRequests:
+                description: |
+                  Настройки максимальных значений CPU и memory, которые может запросить под при выборе узла (если VPA выключен, максимальные значения становятся значениями по умолчанию).
+                properties:
+                  mode:
+                    description: |
+                      Режим управления реквестами ресурсов.
```

```suggestion
                      Режим управления запросами ресурсов.
```

Уместна ли здесь будет такая замена?

---

Да, нормально.